exercises/nucleotide-codons/metadata.toml

```diff
@@ -1,2 +1,2 @@
 title = "Nucleotide Codons"
-blurb = "Write a function that returns the name of an amino acid a particular codon, possibly using shorthand, encodes for."
+blurb = "Return the name of an amino acid encoded by a particular codon, possibly using shorthand."
```

Comment:
It looks like this blurb was overlooked as well.

( Not that I mind &ndash; I actually prefer this one &ndash; but others might. )